### DISCUSSION TITLE
Project status update

### DISCUSSION THREAD

**Discussion by indygreg** [indygreg/PyOxidizer#740] [General]
I haven't meaningfully updated PyOxidizer and other projects in this repository since January 2023.

Background on why is available to read at https://gregoryszorc.com/blog/2024/03/17/my-shifting-open-source-priorities/

tl;dr shifting personal priorities have resulted in me de-prioritizing PyOxidizer (and other open source projects). At this point in time, the future of PyOxidizer is uncertain, possibly dead.

If you are interested in becoming a maintainer, please reply here or shoot me an email (find it in Git commits).

**Comment by jezdez** [comment]
Hey @indygreg, thanks for your transparency and heartfelt message in your blog post, I can relate, truly. 🤗 

Regarding possible help, I'd like to better understand what kind of help would be needed for python-zstandard and if you have a roadmap in mind that I could use to learn a little more where the project is at the moment. Happy to do that here or over email, whatever you prefer.

**Comment by rsyring** [comment]
@indygreg is it time to archive this project so it's clear to people hitting the main repo that the project is dead?  I know you didn't want to admit that in the blog post you wrote, but seems evident now (and that's ok)?
